src/controllers/routers/referees.js

```diff
@@ -89,10 +93,7 @@ const getReferee = async (req, res, next) => {
 const create = async (req, res, next) => {
```

Comment:
I'm getting an error when createdBy or udpatedBy ids are wrapped in ""; For example in the input below, createdBy is accepted but not updatedBy:
 {
        "name": "Ben",
        "email": "Ben5@gmail.com",
        "phone": "88341321",
        "organisation": "UNICEF",
        "createdBy" : 6,
        "updatedBy" : "6"        
    }

Reply:
update to check int in sanitisation using parseInt